framework/src/multiapps/MultiApp.C

```diff
@@ -1149,13 +1149,3 @@ MultiApp::addAssociatedTransfer(MultiAppTransfer & transfer)
 {
   _associated_transfers.push_back(&transfer);
 }
-
-unsigned int
-MultiApp::localAppNumber(const unsigned int global_app) const
-{
-  mooseAssert(
-      hasLocalApp(global_app),
-      "We must have the local app in order for this computation to make sense and be valid");
-
-  return cast_int<unsigned int>(global_app - _first_local_app);
```

Comment:
wasnt this logic to support multi-subapps on each process or something like that?

Reply:
This logic was leveraged when the transfers only had as many coordinate transformation objects as local apps, and so you had to be careful not to index into the transform vector with a global index (that may not be the local index and also may be out of bounds). Now the coordinate transform vector is sized for the number of global apps so that you can always get the transformed position of a multiapp, regardless of whether it is local or not. Consequently we now index into the transform vector with the global index and so we don't have to have a routine for mapping from the global to local index

Reply:
oh yeah sorry I discovered this in the rest of the review and forgot to delete that comment